modules/phase_field/src/kernels/ACInterfaceStress.C

```diff
@@ -0,0 +1,151 @@
+/****************************************************************/
+/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
+/*                                                              */
+/*          All contents are licensed under LGPL V2.1           */
+/*             See LICENSE for full restrictions                */
+/****************************************************************/
+#include "ACInterfaceStress.h"
+#include "RankTwoTensor.h"
+
+template <>
+InputParameters
+validParams<ACInterfaceStress>()
+{
+  InputParameters params = validParams<Kernel>();
+  params.addClassDescription("Interface stress driving force Allen-Cahn Kernel");
+  params.addParam<MaterialPropertyName>("mob_name", "L", "The mobility used with the kernel");
+  params.addParam<std::string>("base_name", "Material property base name");
+  params.addRequiredParam<Real>("stress", "Planar stress");
+  params.addParam<Real>("op_range",
```

Comment:
Perhaps you should put a range check here (positive only)? Mostly so people don't put a zero and cause an error in your initial stress calculation.